app/import-export/import-export.component.html

```diff
@@ -0,0 +1,50 @@
+<h3>
+  You may also like this..
+</h3>
+
+<div class="row">
+  <ng-container *ngIf="newProducts1$ | async as newProducts1">
```

Comment:
I can see that this logic has been repeated 3 times for 3 products, what if u have 10 products?